### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by jonhue** [jonhue/notifications-rails#331] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eduRiveraTipscool** [comment]
Hi all! @jonhue  I have a problem with the gem notification-handler: 

What fails is the rails g notification_handler: install, I have already followed the steps above, and I have tried everything, but nothing, I have the error of: Could not find generator 'notification_handler: install'.
Run `rails generate --help` for more options.

I have looked inside the rails generate --help, and it is not actually created, when I did the bundle install for the gem. I tried to do  spring stop and more things and nothing



Can someone tell me what it is? Is it my problem? or is something from the gem?

**Reply by jonhue** [reply]
Hey, next time please open a new question instead of following up on an existing one.

Are you unable to run the generators of the other gems of this repo or any other gems too? What Ruby and Rails versions are you running?

**Reply by eduRiveraTipscool** [reply]
Oh sorry! okey
@jonhue Not works for any gems of this repo!  i use, Rails 6.0.3.3 - ruby 2.7.0p0

**Reply by jonhue** [reply]
Do any other generators work, for example generating migrations?

**Reply by eduRiveraTipscool** [reply]
yes, all the others generators are working, and i try with other gems and works correctly, the problem that I have is specific for the gem notifications-rails and it content

**Reply by eduRiveraTipscool** [reply]
if you create a new rails project and you put your gem, is working for you? @jonhue
